plugins/imtcp/imtcp.c

```diff
@@ -133,6 +134,8 @@ struct modConfData_s {
 	int iStrmDrvrMode; /* mode for stream driver, driver-dependent (0 mostly means plain tcp) */
 	int iStrmDrvrExtendedCertCheck; /* verify also purpose OID in certificate extended field */
 	int iStrmDrvrSANPreference; /* ignore CN when any SAN set */
+#	define IMTCP_VERIFY_DEPTH 100
```

Comment:
0 makes a better default (see other comment)

Reply:
yes this can be removed now, as default is handeled in nsd_ossl.c now